packages/shared/styles/components/organisms/SfProductCardHorizontal.scss

```diff
@@ -20,6 +20,9 @@
       opacity 150ms ease-in-out
     );
     opacity: var(--product-card-horizontal-image-opacity);
+    &:hover {
```

Comment:
I think we should change `--product-card-horizontal-image-even-opacity` to 1, and default opacity to 0 on image-wrapper hover. `nth-child(even)` is missing `even` in a variable name. Please try this way :) I think there is some problem with size of the image, its coming out of the wrapper, could you look at it also?
![image](https://user-images.githubusercontent.com/34887212/78689349-882a8f80-78f6-11ea-92c8-083c4a59c680.png)

Reply:
http://localhost:6006/?path=/story/organisms-productcard--with-2-pictures at the ProductCard this case looks good :)
you can use this data at the SfProductHorizontal
``` javascript
pictures: [
          {
            mobile: { url: "/assets/storybook/Home/productB.jpg" },
            desktop: { url: "/assets/storybook/Home/productB.jpg" }
          },
          {
            mobile: { url: "/assets/storybook/Home/productA.jpg" },
            desktop: { url: "/assets/storybook/Home/productA.jpg" }
          }
        ]
```

Reply:
pictures changed and if I correctly understood part regarding hover then I don't think there should be opacity 0 on wrapper hover :)